docker/tob-agent/Dockerfile

```diff
@@ -1,4 +1,8 @@
-FROM bcgovimages/aries-cloudagent:py36-1.11-1_0.4.0
+# FROM bcgovimages/aries-cloudagent:py36-1.11-1_0.4.0
+
+FROM bcgovimages/von-image:py36-dev 
+
+RUN pip3 install aries-cloudagent
```

Comment:
@WadeBarnes are we okay using a custom agent image until this fix is included in a new release?
Mostly thinking of the OpenShift deployments that we would need to update.

Reply:
Yes, but please create another ticket so we can track we're using a beta image and need to update it.

Reply:
Created #412 to track this.